src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -220,32 +222,48 @@ public async Task<Manifests> UpdateManifestsAutonomously(Manifests manifests)
                 this.InstallerUrls = installerManifest.Installers.Select(i => i.InstallerUrl).Distinct().ToArray();
             }
 
+            // Generate list of InstallerUpdate objects and parse out any specified architecture overrides.
+            List<InstallerUpdateHelper> installerUpdateHelpers = this.ParseInstallerUrlsForArchOverride(this.InstallerUrls.ToList());
+
+            // If the installer update list is null there was an issue when parsing for architecture override.
+            if (installerUpdateHelpers == null)
+            {
+                return null;
+            }
+
+            // Reassign list with parsed installer urls.
+            this.InstallerUrls = installerUpdateHelpers.Select(x => x.InstallerUrl);

```

Comment:
How could this change from what's passed in?

Reply:
this needs to be reassigned because this.InstallerUrls will have the old installerUrl values with any architecture overrides appended by the user. ParseInstallerUrlsForArchOverride should return the actual installerUrls provided by the user, so this line is needed to make sure this.InstallerUrls is accurate.

Reply:
Thanks for adding the comment!